src/apps/filters/kRing.c

```diff
@@ -59,23 +59,23 @@ int main(int argc, char* argv[]) {
     H3Index origin = 0;
 
     Arg helpArg = {.names = {"-h", "--help"},
-                   .helpText = "Show this help message"};
+                   .helpText = "Show this help message."};
     Arg kArg = {.names = {"-k", NULL},
                 .required = true,
                 .scanFormat = "%d",
                 .valueName = "k",
                 .value = &k,
-                .helpText = "Radius of hexagons"};
+                .helpText = "Radius of hexagons."};
```

Comment:
Nit: Might be clearer as "Radius of ring in hexagons"

Reply:
I'm not sure I understand that since this would produce the neighborhood around the index (cf #42) rather than a hollow ring.

Reply:
Per offline: `Radius in hexagons.`